auth-api/src/auth_api/resources/v1/org.py

```diff
@@ -63,6 +63,8 @@ def search_organizations():
         extract_numbers(request.args.get("id", None)),
         request.args.get("decisionMadeBy", None),
         request.args.get("orgType", None),
+        bool(request.args.get("includeMembers", False)),
+        request.args.get("members", None),
```

Comment:
Added query param (includeMembers) to actually include members payload within an org, by default this is off so there is less data coming back.

Query param "members" is the actual search text used for name or username